docs/testplan/pac/Port_Access_Control.md

```diff
@@ -0,0 +1,562 @@
+
```

Comment:
In Step 16, we don't care the auth type if MAB is disabled.

Reply:
Corrected statement as per review comment.